cmd/runtimecfg/node-ip.go

```diff
@@ -21,7 +21,9 @@ const (
 	nodeIpFile             = "/run/nodeip-configuration/primary-ip"
 	nodeIpIpV6File         = config.NodeIpIpV6File
 	nodeIpIpV4File         = config.NodeIpIpV4File
+	nodeIpNotMatchesVipsFile  = "/run/nodeip-configuration/not-matches-vips"
```

Comment:
Sorry for changing my mind on this, but I think I'd mildly prefer to call the file "remote-worker" instead of "not-matches-vips". It reads better and I think it communicates the intent of the file more clearly.

Reply:
wanted to change it myself, forgot though. Yeah totally agree